isis/src/system/apps/kerneldbgen/kerneldbgen.xml

```diff
@@ -429,7 +432,21 @@
           <item>SMITHEDFILTER</item>
         </exclusions>
       </parameter>
-    </group>
+      <parameter name="SMITHEDPAD">
+        <type>double</type>
+        <brief>
+          Seconds to subtract from the start time and add to the stop time in each smithed kernel.

```

Comment:
This is added to each segment of each kernel, not the entire start and stop time of the kernel.

Reply:
Thanks for catching this. I'll update the documentation.